code/__DEFINES/economy.dm

```diff
@@ -11,6 +11,9 @@
 #define PAYCHECK_COMMAND 200
 #define PAYCHECK_CENTCOM 350
 #define PAYCHECK_CENTCOMCOMMAND 500
+//SKYRAT STUFF, GODDAMIT THOSE FURRIES HAVE COOL STUFF
+#define PAYCHECK_CYCLE_WAIT 3 //How many cycles of SSeconomy firing we're waiting till we get a paycheck. Each cycle is 5
```

Comment:
> Each cycle is 5
5 what? Minutes? Ticks? Please specify.

Reply:
5 minutes, skyrat did have it like that ,heh, going to edit it.